conda.el

```diff
@@ -187,13 +187,18 @@ It's platform specific in that it uses the platform's native path separator."
   (s-trim
    (with-output-to-string
      (with-current-buffer standard-output
-       (process-file shell-file-name nil '(t nil) nil shell-command-switch
-                     (format "%s/bin/conda ..activate \"%s\" %s"
-                             conda-anaconda-home
-                             (if (eq system-type 'windows-nt)
-                                 "cmd.exe"
-                               "bash")
-                             env-dir))))))
+       (let* ((conda-executable-path
+               (concat (file-name-as-directory conda-anaconda-home) (file-name-as-directory "bin") "conda"))
```

Comment:
The error in windows is probably because the conda binary is located at ```conda-anaconda-home/condabin/conda```, not ```conda-anaconda-home/bin/conda```. I am not a frequent user of conda so I don't know if this was a relatively recent change or not.

I'd suggest modifying 191 to be: 
```
               (concat (file-name-as-directory conda-anaconda-home) 
                            (file-name-as-directory (if (eq system-type 'windows-nt) "condabin" "bin")) 
                            "conda"))
```